rfc6962-bis.xml

```diff
@@ -965,7 +983,7 @@ an accepted root certificate.
             </list></t>
 	  </list>
 
-          The <spanx style="verb">hash</spanx> must be calculated as defined in <xref target="tree"/>. The <spanx style="verb">tree_size</spanx> must designate an existing v1 STH.
+          The <spanx style="verb">hash</spanx> must be calculated as defined in <xref target="tree"/>. The <spanx style="verb">tree_size</spanx> must designate an existing v1 STH. Because of skew, the log server may not know the requested STH. In that case, it will return the latest STH it knows, along with an inclusion proof to that STH. If the server knows the requested STH then only <spanx style="verb">leaf_index</spanx> and <spanx style="verb">audit_path</spanx> are returned.
```

Comment:
Nit: replace "requested STH" with "requested tree size".
Also, how about the case when the front-end cannot provide an inclusion proof because the tree size it knows of does not contain the requested leaf? Will it then return the latest STH it knows about or nothing at all?